core/drlibc/drlibc_module_elf.c

```diff
@@ -115,9 +115,9 @@ is_elf_so_header_common(app_pc base, size_t size, bool memory)
             (memory && elf_header.e_ehsize != sizeof(ELF_HEADER_TYPE)) ||
             (memory &&
 #    ifdef X64
-             elf_header.e_machine != EM_X86_64 && elf_header.e_machine != EM_AARCH64
+             elf_header.e_machine != IF_AARCHXX_ELSE(EM_AARCH64, EM_X86_64)
```

Comment:
This breaks cross-arch support where we want to map AArch64 libraries on an x86 machine (basically this undoes recent changes to this code).  Probably best to instead add a define for `EM_AARCH64` if it's not already defined to module_elf.h.

Reply:
Maybe should check `#ifdef EM_AARCH64` and add code if defined?

Reply:
It is not defined in `module_elf.h`. We can defined it there if it is a known constant.

Reply:
So, what would be the best choice?

Reply:
I think you must have missed the 2nd sentence of my comment?

Reply:
Sorry, I will add a define to module_elf.h.

Reply:
Done.